2025/05/07 07:52:34 [notice] 3414#3414: using inherited sockets from "6;7;"
2025/05/07 09:43:07 [notice] 3843#3843: signal process started
2025/05/07 11:34:53 [notice] 4288#4288: signal process started
2025/05/07 11:34:59 [error] 4289#4289: *26 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/07 11:35:02 [error] 4289#4289: *26 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/07 11:38:56 [notice] 4315#4315: signal process started
2025/05/07 16:10:24 [notice] 4118#4118: signal process started
2025/05/07 16:51:39 [notice] 4495#4495: signal process started
2025/05/08 05:09:18 [error] 679#679: *1 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/08 05:10:03 [error] 679#679: *1 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/08 05:10:06 [error] 679#679: *1 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/08 05:10:08 [error] 679#679: *1 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/08 05:10:10 [error] 679#679: *1 connect() failed (111: Unknown error) while connecting to upstream, client: 122.173.31.204, server: ai-log-analyser.eastasia.cloudapp.azure.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5000/", host: "ai-log-analyser.eastasia.cloudapp.azure.com"
2025/05/08 18:48:35 [alert] 604#604: *39 open socket #3 left in connection 4
2025/05/08 18:48:35 [alert] 604#604: aborting
